docs/dev/rfc_process.md

```diff
@@ -105,5 +105,46 @@ While we encourage and celebrate every contributor, the bar for RFC acceptance i
 *   Failure to achieve consensus during the design review.
 *   Concerns raised during implementation (for example: inability to achieve backwards compatibility, concerns about maintenance).
 
-If this process is functioning well, RFCs are expected to fail in the earlier, rather than later stages. 
+If this process is functioning well, RFCs are expected to fail in the earlier, rather than later stages.
 An approved RFC is not a commitment to implementation on any sort of timeline. The prioritization of features depends on user interest and willingness of contributors to implement them.
+
+## New hardware integrations
+
+Several hardware vendors already have integrations with cirq.  We are not
+currently soliciting additional vendors.  However, if you are considering
+integrating with cirq, we would highly encourage you to engage with the
+cirq-maintainer team through attending the weekly cirq cync and submitting
+an RFC as specified above.  Everyone benefits from a well-maintained, user
```

Comment:
do we have an example of a vendor integration that would be useful to link to?

Reply:
Linked to hardware pages.  I was reluctant to pick one in particular.